src/components/Header/Header.style.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
```

Comment:
The header doesn't match the design in Zeplin yet (see below). Have a look at the [knowledge base](https://www.notion.so/Pixel-perfect-layouts-5b37e3c869cf45a4bf05736d94d75394) for more information about pixel-perfect layouts. The height should be 100px, the horizontal margin 80px. 

I think a max-width of 1440px for the whole page would be appropriate, too. Maybe I should mention that in the ticket since it's not clear from the design.

![image](https://user-images.githubusercontent.com/9722792/77728099-1439de80-6ffc-11ea-8c6d-b7c378ba1664.png)

Reply:
I noticed I hadnt done yesterday when I went through the pixel perfect page and worked on it last night. It will be part of my Footer PR. I will give my thoughts via Slack a little bit later as I feel there is an improvement that could be made.

Reply:
:+1: